### DISCUSSION TITLE
关于页面中“所有代码都是这一个人敲的”是否不妥

### DISCUSSION THREAD

**Discussion by idhaname** [Meloong-Git/PCL#3487] [提问] [answered]
![图片](https://github.com/Hex-Dragon/PCL2/assets/68469786/17fc5fec-10e5-4393-bf4f-6f341e9f377a)
如题，自从PCL2部分代码开源后，就有很多贡献者提交了PR、Issues等，为完善PCL2做了许多贡献
因此，请问关于页面中“所有代码都是这一个人敲的”文案是否需要更换，
是否也可以考虑下在特别鸣谢中加入“所有帮助完善PCL2的贡献者”之类的…

**Comment by Pigeon0v0** [comment]
目前来说大家的 PR 改动都是文字层面的...

**Reply by iPodtouch6557** [reply]
没有专业的开发者看看底层代码

**Comment by Silverteal** [comment, marked as answer]
PCL不是传统意义上的开源软件（因为没有使用开源许可证）。目前为止，此仓库的绝大部分除所有者以外的Commit都不是代码。少量的代码也几乎全部是文字方面的修改。说是“所有代码都是一个人敲的”虽不完全严谨，但也与事实无太大出入。于我而言，可以不替换。

对于“贡献者”，开源社区一般定义为“提交过Commit的人”，因此仅提出Issue、Discussion一般不足以视为贡献者。我认为特别鸣谢的“内群用户”这一栏暂时足以覆盖已有贡献者，也可以不替换。

~~替换也很简单，交个PR就好了~~
这段好像没有开源
